src/testdrive/src/action.rs

```diff
@@ -531,8 +531,14 @@ impl Run for PosCommand {
         };
 
         let r = match self.command {
-            Command::Builtin(mut builtin, version_constraint) => {
-                handle_version!(version_constraint);
+            Command::Builtin(mut builtin, modifiers) => {
+                handle_version!(modifiers.version_constraint);
+                if modifiers.retry == Some(true) {
```

Comment:
`[retry]` also needs to be documented in `testdrive.md` . Any existing references to automatic retrying should be reviewed/removed.